docs/execution/dataloader.rst

```diff
@@ -111,8 +111,10 @@ leaner code and at most 4 database requests, and possibly fewer if there are cac
         best_friend = graphene.Field(lambda: User)
         friends = graphene.List(lambda: User)
 
-        def resolve_best_friend(self, info):
-            return user_loader.load(self.best_friend_id)
+        @staticmethod
+        def resolve_best_friend(root, info):
```

Comment:
```suggestion
        def resolve_best_friend(user, info):
```

`root` has a specific meaning in graphql (it’s the value that is passed into the root of the tree) and so anything that is it the `Query` object should use variable names that indicate what the variable is, so `user` in this case.